### PR TITLE
Update DistSQL documents

### PR DESCRIPTION
**Pull request by Qianyi951015**
Changes proposed in this pull request:
  - Uniform Railroad Diagram Format in DistSQL documents
  - Correct some errors in Railroad Diagram

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md [hunk 1]
```diff
@@ -17,10 +17,13 @@ lableName ::=
...
 trafficAlgorithmDefination ::=
-  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' = trafficAlgorithmTypeName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' '=' trafficAlgorithmTypeName (propertiesDefination)? ')' ')'
 
 loadBalancerDefination ::=
-  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' = loadBalancerName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' '=' loadBalancerName (propertiesDefination)? ')' ')'
+
+propertiesDefination ::=
+  ',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')'
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md:26]: Hi @Qianyi951015 ,  I think a definition should not start with a `comma`, do you agree?

File: docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.en.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ shardingSize ::=
...
+  'RATE_LIMITER' '(' 'TYPE' '(' 'NAME' '=' algorithmName ',' 'PROPERTIES' '(' propertiesDefinition ')' ')' ')'
 
 algorithmName ::=
   string
 
-propertyDefinition ::=
-  ( key  '=' value ) ( ',' key  '=' value )* 
-
+propertiesDefinition ::=
+  ',' 'PROPERTIES' '(' ( key  '=' value ) ( ',' key  '=' value )* ')'
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.en.md:41]: Conflicts with line 35

File: docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md (hunk 1 above)
Comment (Qianyi951015) [reply docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md:26]: Change like this better?
```
'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' '=' loadBalancerName (',' propertiesDefination)? ')' ')'
propertiesDefination ::=
   'PROPERTIES' '(' key '=' value (',' key '=' value)* ')'
```

File: docs/document/content/reference/distsql/syntax/rdl/rule-definition/db-discovery/alter-db-discovery-rule.cn.md [hunk 3]
```diff
@@ -27,6 +30,12 @@ typeName ::=
 
 discoveryHeartbeatName ::=
   identifier
+
+key ::=
+  string
+
+value ::=
+  string
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/rdl/rule-definition/db-discovery/alter-db-discovery-rule.cn.md:38]: Sorry, `value` is not a string, it is a literal now, please refer to #22891

File: docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md (hunk 1 above)
Comment (RaigorJiang) [reply docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md:26]: Yes.

File: docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.cn.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ writeConfiguration ::=
   'WRITE' '(' ('WORKER_THREAD' '=' workerThreadPoolSize ',')? ('BATCH_SIZE' '=' batchSize ',')? ('SHARDING_SIZE' '=' shardingSize ',')? (rateLimiter)? ')'
 
 dataChannel ::=
-  'STREAM_CHANNEL' '(' 'TYPE' '(' 'NAME' '=' algorithmName ',' 'PROPERTIES' '(' propertyDefinition ')'
+  'STREAM_CHANNEL' '(' 'TYPE' '(' 'NAME' '=' algorithmName propertyDefinition
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.cn.md:23]: `propertyDefinition` is removed at line 40, and comma missed after algorithmName.

File: docs/document/content/reference/distsql/syntax/ral/global-rule/create-traffic-rule.cn.md [hunk 5]
```diff
@@ -17,10 +17,13 @@ lableName ::=
...
 trafficAlgorithmDefination ::=
-  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' = trafficAlgorithmTypeName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' '=' trafficAlgorithmTypeName (propertiesDefination)? ')' ')'
 
 loadBalancerDefination ::=
-  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' = loadBalancerName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' '=' loadBalancerName (propertiesDefination)? ')' ')'
+
+propertiesDefination ::=
+  ',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')'
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/global-rule/create-traffic-rule.cn.md:26]: Same problem

File: docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.en.md [hunk 6]
```diff
@@ -32,19 +32,19 @@ shardingSize ::=
...
 
 algorithmName ::=
   string
 
-propertyDefinition ::=
-  ( key  '=' value ) ( ',' key  '=' value )* 
-
+propertiesDefinition ::=
+  'PROPERTIES' '(' key '=' value (',' key '=' value)* ')'
+
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/migration/alter-migration-rule.en.md:42]: Useless white space at line 42.

File: docs/document/content/reference/distsql/syntax/ral/migration/register-migration-source-storage-unit.en.md [hunk 7]
```diff
@@ -28,17 +28,14 @@ user ::=
 password ::=
   string
 
-proerties ::=
-  PROPERTIES '(' property ( ',' property )* ')'
-
-property ::=
-  key '=' value
+proertiesDefinition ::=
+  ',' 'PROPERTIES' '(' ( key  '=' value ) ( ',' key  '=' value )* ')'
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/migration/register-migration-source-storage-unit.en.md:32]: need to be updated.

File: docs/document/content/reference/distsql/syntax/rdl/storage-unit-definition/alter-storage-unit.en.md [hunk 8]
```diff
@@ -37,11 +37,8 @@ user ::=
 password ::=
   string
 
-proerties ::=
-  PROPERTIES '(' property ( ',' property )* ')'
-
-property ::=
-  key '=' value
+proertiesDefinition ::=
+  ',' PROPERTIES '(' key '=' value (',' key '=' value)* ')'
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/rdl/storage-unit-definition/alter-storage-unit.en.md:41]: need to be updated

File: docs/document/content/reference/distsql/syntax/rdl/storage-unit-definition/register-storage-unit.en.md [hunk 9]
```diff
@@ -37,11 +37,8 @@ user ::=
 password ::=
   string
 
-proerties ::=
-  PROPERTIES '(' property ( ',' property )* ')'
-
-property ::=
-  key '=' value
+proertiesDefinition ::=
+  ',' PROPERTIES '(' key '=' value (',' key '=' value)* ')'
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/rdl/storage-unit-definition/register-storage-unit.en.md:41]: need to be updated

File: docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md [hunk 10]
```diff
@@ -17,10 +17,13 @@ lableName ::=
...
 
 trafficAlgorithmDefination ::=
-  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' = trafficAlgorithmTypeName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'TRAFFIC_ALGORITHM' '(' 'TYPE' '(' 'NAME' '=' trafficAlgorithmTypeName (',' propertiesDefination)? ')' ')'
 
 loadBalancerDefination ::=
-  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' = loadBalancerName (',' 'PROPERTIES' '(' key '=' value (',' key '=' value)* ')')? ')' ')'
+  'LOAD_BALANCER' '(' 'TYPE' '(' 'NAME' '=' loadBalancerName (',' propertiesDefination)? ')' ')'
+
+propertiesDefination ::=
```
Comment (RaigorJiang) [inline docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md:25]: Should it be Definition?

Comment (Qianyi951015) [reply docs/document/content/reference/distsql/syntax/ral/global-rule/alter-traffic-rule.cn.md:25]: yes😂

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**